cfgov/cfgov/settings/base.py

```diff
@@ -376,7 +376,10 @@
 
 # Email
 ADMINS = admin_emails(os.environ.get('ADMIN_EMAILS'))
-EMAIL_SUBJECT_PREFIX = os.environ.get('EMAIL_SUBJECT_PREFIX')
+
+if DEPLOY_ENVIRONMENT:
+    EMAIL_SUBJECT_PREFIX = u'[{}] '.format(DEPLOY_ENVIRONMENT)
+
```

Comment:
[The default is just `[Django]`](https://docs.djangoproject.com/en/1.8/ref/settings/#email-subject-prefix), which is less than ideal but no worse than we have now. I think that's acceptable.